rocAL/rocAL_pybind/amd/rocal/plugin/tf.py

```diff
@@ -155,7 +155,7 @@ def __next__(self):
         elif (self.loader._name == "TFRecordReaderClassification"):
             if(self.loader._oneHotEncoding == True):
                 self.labels = np.zeros((self.bs)*(self.loader._numOfClasses),dtype = "int32")
-                self.loader.GetOneHotEncodedLabels_TF(self.labels)
+                self.loader.GetOneHotEncodedLabels(self.labels, device="cpu")
```

Comment:
why was this changed?

Reply:
A change from ToT not reflected in Shobana's fork